packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -77,6 +80,7 @@
   private int currentOrientation = ORIENTATION_UNKNOWN;
   private FlashMode flashMode;
   private PictureCaptureRequest pictureCaptureRequest;
+  private static HashMap<String, Integer> supportedImageFormats;
```

Comment:
You can define the map and all values statically in Java using the following syntax:

```suggestion
  private static HashMap<String, int> supportedImageFormats;
  static {
      supportedImageFormats = new HashMap<>();
      supportedImageFormats.put("yuv420", 35);
      supportedImageFormats.put("jpeg", 256);
  }
```

or if you prefer a bit more semantic (and don't want to use the anonymous class) you could use:

```java
  private static HashMap<String, int> supportedImageFormats = createSupportedImageFormats();
  private Map<String, int> createSupportedImageFormats() {
    final Map<String, int> supportedFormats = new HashMap<>();
    supportedFormats.put("yuv420", 35);
    supportedFormats.put("jpeg", 256);
    return supportedFormats;
  }
```